src/openforms/config/models/config.py

```diff
@@ -321,15 +321,18 @@ class GlobalConfiguration(SingletonModel):
     )
 
     # global payment settings
-    payment_order_id_prefix = models.CharField(
-        _("Payment Order ID prefix"),
-        max_length=16,
-        default="{year}",
-        blank=True,
-        help_text=_(
-            "Prefix to apply to generated numerical order IDs. Alpha-numerical only, supports placeholder {year}."
-        ),
-        validators=[validate_payment_order_id_prefix],
+    payment_order_id_template = models.CharField(
+        _("Payment Order ID template"),
+        max_length=48,
+        default="{year}/{reference}/{uid}",
```

Comment:
```suggestion
        default="{year}/{public_reference}/{uid}",
```
I'd really prefer using the same variable names consistently across the project. All other templates use `public_reference`.